src/js/utils.js

```diff
@@ -107,8 +107,13 @@ this.exports = {
   },
   
   hasSensors: function(thermostat) {
-    return true; //TODO: Fixme!
+    var hasSensors = false;
+    thermostat.remoteSensors.forEach(
+      function(sensor) {
+          if (sensor.type ==='ecobee3_remote_sensor') {
```

Comment:
The thermostats internal sensors are in this array but if ecobee considers them sensors then the app probably should too. I will update to your suggestion.